redfish-core/lib/hypervisor_system.hpp

```diff
@@ -1052,7 +1052,7 @@ inline void requestRoutesHypervisorSystems(App& app)
 
     BMCWEB_ROUTE(app,
                  "/redfish/v1/Systems/hypervisor/EthernetInterfaces/<str>/")
-        .privileges(redfish::privileges::patchOEMHypervisorEthInterface)
+        .privileges({{"ConfigureManager"}})
```

Comment:
Ideally this is a separate PR..  And it includes the comment 
 // Restrict the hypervisor ethernet interface PATCH to ConfigureManager

Reply:
Should I revert it back?

Reply:
Reverted the changes